exonum-java-proofs/src/test/java/com/exonum/binding/storage/proofs/map/flat/CheckedFlatMapProofTest.java

```diff
@@ -0,0 +1,123 @@
+package com.exonum.binding.storage.proofs.map.flat;
+
+import static com.exonum.binding.hash.Funnels.hashCodeFunnel;
+import static com.exonum.binding.storage.proofs.DbKeyFunnel.dbKeyFunnel;
+import static com.exonum.binding.test.Bytes.bytes;
+import static com.exonum.binding.test.Bytes.createPrefixed;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import com.exonum.binding.hash.HashCode;
+import com.exonum.binding.hash.HashFunction;
+import com.exonum.binding.hash.Hashing;
+import com.exonum.binding.storage.proofs.map.DbKey;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+
+public class CheckedFlatMapProofTest {
```

Comment:
I'd rename it into **Un**checked as it tests mostly that class — this way navigation will work properly!